.eslintrc.js

```diff
@@ -1,23 +1,38 @@
 module.exports = {
-    "env": {
-        es6: true
-    },
-    "parser": "babel-eslint",
-    "extends": 'airbnb',
-    "plugins": [
-        "react"
+  "env": {
+    es6: true,
+    browser: true
+  },
+  "parser": "babel-eslint",
+  "extends": 'airbnb',
+  "plugins": [
+    "react"
+  ],
+  rules: {
+    'consistent-return': 0,
+    'no-console': 0,
+     // Look into this - forbidding object & array could be useful, but
+     // it leads to repetitions of object shapes for props passed through
+     // several components
+    'react/forbid-prop-types': 0,
+    'react/jsx-filename-extension': [2, { "extensions": [".js"] }],
+    'react/jsx-no-bind': [
+      2,
+      {
+        allowArrowFunctions : false,
+        allowBind: false,
+        ignoreRefs: false,
+      }
     ],
-    rules: {
-        'consistent-return': 0,
-        'no-console': 0,
-        'react/jsx-no-bind': [
-          2,
-          {
-            allowArrowFunctions : false,
-            allowBind: false,
-            ignoreRefs: false,
-          }
-        ],
-        semi: [2, 'never']
-    }
+    'react/sort-comp': [2, {
+      order: [
+        'static-methods',
+        'lifecycle',
+        'everything-else',
+        'render'
+      ]
+    }],
+    'jsx-a11y/accessible-emoji': 0,
```

Comment:
Yep, it's a jsx-a11y check activated by the config we inherit from eslint-config-airbnb, except I guess the presets are mostly for web React ^^'